util/topsql/topsql.go

```diff
@@ -120,7 +122,7 @@ func linkSQLTextWithDigest(sqlDigest []byte, normalizedSQL string, isInternal bo
 		normalizedSQL = normalizedSQL[:MaxSQLTextSize]
 	}
 
-	c := tracecpu.GlobalSQLCPUProfiler.GetCollector()
+	c := tracecpu.GlobalSQLCPUCollector.GetCollector()
```

Comment:
Weired... '`GetCollector`', change it dynamicly to `TopSQLReporter` for calling `RegisterSQL` and `RegisterPlan`.

Reply:
great, resolved.